packages/mdc-button/_mixins.scss

```diff
@@ -164,10 +164,16 @@
 }
 
 @mixin mdc-button-shape-radius($radius, $rtl-reflexive: false) {
-  @include mdc-shape-radius(mdc-shape-resolve-percentage-radius($mdc-button-height, $radius), $rtl-reflexive);
+  @include mdc-shape-radius(
```

Comment:
Store first parameter into a local variable instead?

```
$resolved-radius: mdc-shape-resolve-percentage-radius($mdc-button-height, $radius);
@include mdc-shape-radius($resolved-radius, $rtl-reflexive);
```

Same below.

Reply:
TBH I'm not even sure why these lines changed, since we typically fit to 120 width. (Though now that I'm checking that's only in our eslintrc, not stylelint)

Reply:
whoops sorry about that, was experimenting with using prettier and I guess it went crazy